components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -56,16 +56,18 @@
           <div class ="channel-line">
             <chef-badge *ngIf="service.channel !== ''">{{ service.channel | uppercase }}</chef-badge>
             <chef-badge no-data
-              id="general-channel-tooltip"
+              [id]="'channel-tooltip-' + i"
               [tooltip]="tooltipMessageFor('channel')"
               *ngIf="service.channel === ''">NO CHANNEL</chef-badge>
           </div>
           <div class ="health-line">
             <chef-icon class="{{ service.health_check }}">{{ service.health_check | serviceStatusIcon }}</chef-icon>
-            {{ healthCheckStatus(service) }}
+            {{ healthCheckStatus(service.health_check) }}
           </div>
-          <div class ="timewizard-line">
-            Since {{ service.current_health_since }} ago
+          <chef-tooltip [attr.for]="'health-timestamp' + i">{{ formatTimestamp(service.health_updated_at) }}</chef-tooltip>
+          <div class="timewizard-line">
+            {{ timewizardMessage(service.health_check, service.previous_health_check) }}
+            <a [id]="'health-timestamp' + i"> {{ service.current_health_since }} ago</a>
```

Comment:
Nitpick: should this be `'health-timestamp-'`?

Reply:
changed